src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -742,7 +742,7 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         ///  [!code-csharp[FastTree](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/RandomTrainerSample.cs)]
         /// ]]></format>
```

Comment:
If you want to remove the sample I think you also need to remove the reference here.